app/Controller/PagesController.php

```diff
@@ -74,3 +120,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
```

Comment:
__[code_sniffer]__
* Inline comments must end in full-stops, exclamation marks, or question marks